jobs/transforms/audits_registry.py

```diff
@@ -41,3 +42,16 @@ def get_audit_do_fn_types_by_kind():
         targeted.
     """
     return audit_decorators.AuditsExisting.get_audit_do_fn_types_by_kind()
+
+
+def get_property_relationships_by_kind():
+    """Returns the property relationships between models.
+
+    Returns:
+        dict(str: dict(str: tuple(str))). Property relationships keyed by the
```

Comment:
It's not completely clear from the description what the second str represents. I suggest being more explicit: "For each model kind, the corresponding value is a dict whose keys are XXX and whose corresponding values are YYY."

Reply:
Obsolete.